File: easybluetoothble/src/main/java/com/srise/easybluetoothble/Interface/IMessageTool.java
package com.srise.easybluetoothble.Interface;


public interface IMessageTool {
    byte[] encodeData(int newProtocolType, int requestId, int data);
    byte[] encodeData(int newProtocolType, int requestId, String data);
    byte[] encodeData(int newProtocolType, int requestId, byte[] data);
}

File: ktlionble/src/main/java/c/example/dell/ktlionble/inter/IMessageTool.kt
package c.example.dell.ktlionble.inter

interface IMessageTool {
    fun encodeData(newProtocolType: Int, requestId: Int, data: Int): ByteArray?
    fun encodeData(newProtocolType: Int, requestId: Int, data: String?): ByteArray?
    fun encodeData(newProtocolType: Int, requestId: Int, data: ByteArray?): ByteArray?
}